src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -171,6 +171,11 @@ image::execute.gif[100000,2000]
 The ActiveEon Jupyter Kernel adds a kernel backend to Jupyter. This kernel interfaces directly with the ProActive
 scheduler and constructs tasks and workflows to execute them on the fly.
 
+In seconds, you can now switch from executing your code on your local machine and on a remote server. Develop locally
```

Comment:
Honestly, I tried to refactor and understand this paragraph but I couldn't. please refactor or fix.